packages/libplist.rb

```diff
@@ -16,10 +16,9 @@ class Libplist < Package
 
   def self.build
     system './autogen.sh CC=gcc CXX=g++'
-    system './configure',
-      "--prefix=#{CREW_PREFIX}",
-      "--libdir=#{CREW_LIB_PREFIX}",
-      '--disable-dependency-tracking'
+    system "./configure \
+      --prefix=#{CREW_PREFIX} \
+      --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
I tried this and it doesn't work either.  It appears none of the gcc header files are included while attempting to compile.

Reply:
On which arch and what's the output of `crew list installed`?

Reply:
armv7l: See below:
```
acl
alien
alsa_lib
alsa_plugins
alsa_utils
apr
apriconv
aspell
aspell_en
at_spi2_atk
at_spi2_core
atk
attr
autoconf
automake
binutils
bison
brotli
bz2
cairo
chromaprint
cloog
cmake
codium
composer
compressdoc
cras
cups
curl
dav1d
dbus
dbus_glib
diffutils
docbook
docbook_xml
docbook_xsl
doxygen
ecasound
elfutils
eudev
exif
expat
ffmpeg
filecmd
flac
flex
fltk
font_util
fontconfig
fop
freeglut
freetds
freetype
freetype_sub
fribidi
fskit
fuse
gawk
gcc8
gcc_tools
gdbm
gdk_pixbuf
geoip
gettext
gexiv2
git
glib
glib_networking
glibc
glproto
gmp
gnome_common
gnome_icon_theme
gnutls
go_bootstrap
gobject_introspection
gperf
gphoto
graphene
graphviz
groff
gsettings_desktop_schemas
gsl
gstreamer
gtk2
gtk3
gtk_doc
harfbuzz
hicolor_icon_theme
hunspell_base
hunspell_en_us
icon_naming_utils
icu4c
ilmbase
iniparser
intltool
isl
iso_codes
jasper
jdk8
json_glib
jsonc
jsoncpp
krb5
ladspa
lcms
ld_default
less
libass
libaudiofile
libavc1394
libbsd
libcap
libcap_ng
libcheck
libcroco
libcyrussasl
libdc1394
libdmx
libdrm
libedit
libepoxy
libevdev
libevent
libexif
libexiv2
libfdk_aac
libffi
libfontenc
libfrei0r
libfs
libgconf
libgcrypt
libglu
libgpgerror
libgphoto
libgudev
libice
libiconv
libidn2
libinput
libjpeg
libjpeg_turbo
libmetalink
libmp3lame
libnghttp2
libnl3
libogg
libopencoreamr
libopus
libpcap
libpciaccess
libpipeline
libpng
libproxy
libpsl
libpstat
libraw1394
libressl
librsvg
libsdl
libsdl2
libsecret
libsigsegv
libsm
libsndfile
libsodium
libsoup
libsoxr
libssh
libssh2
libtasn1
libtheora
libtiff
libtirpc
libtool
libunbound
libunistring
libunwind
libusb
libva
libvdpau
libvoamrwbenc
libvorbis
libvpx
libwacom
libwebp
libx11
libx264
libx265
libxau
libxaw
libxcb
libxcomposite
libxcursor
libxdamage
libxext
libxfixes
libxfont2
libxft
libxi
libxinerama
libxkbcommon
libxkbfile
libxml2
libxml2_python
libxmu
libxpm
libxrandr
libxrender
libxres
libxscrnsaver
libxshmfence
libxslt
libxt
libxtrans
libxtst
libxv
libxvid
libxvmc
libxxf86dga
libxxf86vm
libzip
linuxheaders
llvm
lvm2
lz4
lzip
lzo
m4
make
mandb
manpages
mesa
meson
most
mpc
mpfr
mtdev
ncurses
nettle
ninja
nodebrew
oniguruma
openal
openexr
openjpeg
openldap
openpam
osl
p11kit
pango
patch
pcre
pcre2
perl
perl_locale_messages
perl_text_unidecode
perl_unicode_eastasianwidth
perl_xml_parser
perl_xml_simple
pixman
pkgconfig
platformsh
poppler
popt
protobuf
psmisc
pthread_stubs
pugixml
pulseaudio
python27
python3
qrencode
qtbase
re2c
readline
rtmpdump
ruby
sane_backends
sbc
sed
setuptools
sgml_common
shared_mime_info
six
slang
sommelier
speex
speexdsp
sqlite
swig
tcl
tcpwrappers
tdb
testdisk
texinfo
tidy
trousers
uchardet
unixodbc
unzip
util_linux
util_macros
vala
vdev
vidstab
vpnc
wavpack
wayland
wayland_protocols
wget
xbitmaps
xcb_proto
xcb_util
xdg_base
xkbcomp
xkeyboard_config
xorg_proto
xorg_server
xsetroot
xzutils
yarn
yasm
zeromq
zip
zlibpkg
zvbi
```

Reply:
On a fresh install I have no problem compiling it. What else is different between an aarch64 and armv7 machine? Should we do a diff of `/usr/local/lib` on a fresh install on both machines and compare?

Reply:
I have the exact same issue for x86_64.  Can you try and see if it compiles for you on that architecture?

Reply:
See this issue as I believe we're running into the same problem: https://github.com/libimobiledevice/usbmuxd/issues/108

I just did a fresh install and was able to build this package from source successfully. I think over time as I test PRs I end up with junk that's either not being removed properly or leaving mismatched versions of some libraries around.

Reply:
I can build on x86_64 successfully on a fresh install in Docker. I don't have any real x86_64 chromebook hardware to test on.